docs/topics/api/reviews.rst

```diff
@@ -114,6 +115,7 @@ If successful a :ref:`review object <review-detail-object>` is returned.
     :<json string|null body: The text of the review.
     :<json string|null title: The title of the review.
     :<json int rating: The rating the user wants to give as part of the review.
+    :<json int version: The add-on version id the review applies to.
```

Comment:
Was this just an omission in the docs? Did it always require you to post a version ID?